client/components/TestQueueRow/index.jsx

```diff
@@ -517,39 +517,30 @@ const TestQueueRow = ({
             );
         } else if (runStatus === 'DRAFT' || !runStatus) {
             status = <span className="status-label not-started">Draft</span>;
-        } else if (runStatus === 'IN_REVIEW') {
-            status = (
-                <span className="status-label in-progress">In Review</span>
-            );
-        }
+        } 
 
         return { status, results };
     };
 
     const evaluateNewReportStatus = () => {
-        const { status, conflictsLength } = testPlanReport;
+        const { conflictsLength } = testPlanReport;
         const conflictsCount = conflictsLength;
```

Comment:
nit: I think the language was being preserved based on an earlier iteration of the application but the `conflictsCount` variable may be unnecessary as it isn't (and cannot be) mutated, so we could reference `conflictsLength` directly. We can maybe add a comment that it is the `count` as well if we'd like to be explicit.

This also applies to its usage in `evaluateStatusAndResults`